File: java/MaratonaJava/javinha/src/main/java/com/maratonajava/estrutura_sequencial/exerc04.java
package com.maratonajava.estrutura_sequencial;

import java.util.Scanner;

public class exerc04 {
    //Faça um Programa que peça as 4 notas bimestrais e mostre a média. 
    public static void main(String[] args){
        try(Scanner notas = new Scanner(System.in)){
            float nota1,nota2,nota3,nota4,mediaNota;

            System.out.println("Programa Peça s notas e mostre a Média.");
            System.out.println("================================================");

            System.out.println("Digite o valor da sua 1° Nota.");
            nota1 = notas.nextFloat();

            System.out.println("Digite o valor da sua 2° Nota.");
            nota2 = notas.nextFloat();

            System.out.println("Digite o valor da sua 3° Nota.");
            nota3 = notas.nextFloat();

            System.out.println("Digite o valor da sua 4° Nota.");
            nota4 = notas.nextFloat();

            mediaNota = (nota1+nota2+nota3+nota4)/4;

            System.out.println("Sua Média é "+mediaNota);
        }
    } 
}

File: C/Exercicios/Estrutura_Sequencial/exerc04.c
#include <stdio.h>

#define descricao_programa "Faça um Programa que peça as 4 notas bimestrais e mostre a média.\n"

int main(){
    printf("%s",descricao_programa);

    float media_nota, nota1, nota2,nota3, nota4;

    printf("Informe o valor da 1° nota.\n");
    scanf("%f",&nota1);

    printf("Informe o valor da 2° nota.\n");
    scanf("%f", &nota2);

    printf("Informe o valor da 3° nota.\n");
    scanf("%f", &nota3);

    printf("Informe o valor da 4° nota.\n");
    scanf("%f", &nota4);

    media_nota = (nota1+nota2+nota3+nota4)/4;

    printf("Sua média é: %f.\n", media_nota);
}